net/proxy_resolution/proxy_config_service_tor.cc

```diff
@@ -52,59 +49,70 @@ TorProxyConfigService::TorProxyConfigService(
       }
       if (scheme_.empty() || host_.empty() || port_.empty())
         return;
-      std::string proxy_url;
-      if (tor_proxy_map && !username.empty()) {
-        std::string password = tor_proxy_map->Get(username);
-        proxy_url = std::string(scheme_ + "://" + username + ":" + password +
-                                "@" + host_ + ":" + port_);
-      } else {
-        proxy_url = std::string(scheme_ + "://" + host_ + ":" + port_);
-      }
+      std::string proxy_url =
+        std::string(scheme_ + "://" + host_ + ":" + port_);
```

Comment:
whitespace could be fixed by `clang-format`

Reply:
nit: use `const std::string` and generally `const` as much as possible

Reply:
fixed in 4985fcefcc3a24b6706b065df1d5f26af54b3ac0